Winapp3/Winapp3.ini

```diff
@@ -629,10 +629,11 @@ FileKey13=%SystemDrive%\GOG Games\*|*.log;*.tmp|RECURSE
 FileKey14=%SystemDrive%\GOG Games\*\logs|*.*|RECURSE
 FileKey15=%SystemDrive%\GOG Games\*\_Redist|*.*|RECURSE
 
-[Junk File Extensions *]
+[Junk File Extensions Superfixed*]

```

Comment:
Why?

Reply:
Sorry I have logged in now, I wanted to change it back to the original name.
I realy forgot to rename it back.

Sorry

Reply:
That's fine. I figured it was a mistake.